### PR TITLE
Finnish locale for customBlockManager

### PR DESCRIPTION
**Pull request by ajnyga**
Hi @mtub 

This is the fi locale for the plugin.

### CODE REVIEW

File: locale/fi_FI/locale.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../../lib/pkp/dtd/locale.dtd">
+
+<!--
+  * plugins/generic/customBlockManager/locale/en_US/locale.xml
```
Comment (mtub) [inline locale/fi_FI/locale.xml]: path should be fi_FI...

File: locale/fi_FI/locale.xml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  *
+  * Copyright (c) 2014-2016 Simon Fraser University Library
```
Comment (mtub) [inline locale/fi_FI/locale.xml]: 2014-2017

File: locale/fi_FI/locale.xml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  * Copyright (c) 2003-2016 John Willinsky
```
Comment (mtub) [inline locale/fi_FI/locale.xml]: 2003-2017

**Comment by ajnyga** [comment]
thanks and sorry, I promise to be a bit more careful next time with these...
See the new commit...

**Pull request merged**